example/android/build.gradle

```
@@ -24,7 +24,7 @@ buildscript {
         classpath("com.android.tools.build:gradle:7.2.1")
         classpath("com.facebook.react:react-native-gradle-plugin")
         classpath("de.undercouch:gradle-download-task:5.0.1")
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
```

Just to keep the convention.

```suggestion
        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
```